doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,93 @@
+
+The ambassador program does that by:
+
+* Maintaining an up-to-date set of messages that it would like ambassadors
+  to promote. These will be documented in the nodejs/ambassadors repository.
+* Maintaining an up-to-date set of topics on which additional content
+  would benefit the community. These will be documented in the
+  nodejs/ambassadors repository.
+* Providing a space for collaborators to share new topics/features they
+  are working on, when they would be happy for ambassadors to help share and get
+  early feedback.
+* Helping ambassadors promote content that aligns with the
+  messages and topics defined through social media and other channels.
+* Advocating for ambassadors to be part of the OpenJS speakers bureau, even if the
+  ambassador is not otherwise an active member of the project itself.
+
+## Ambassadors nominations
+
+Any member of the Node.js organization may nominate an ambassador.
+The Technical Steering Committee (TSC) will review/approve ambassadors.
+
+In the initial phase of the program:
+
+* there will be a maximum of 4 ambassadors
+* the term will be for the calendar year. After a year ambassadors automatically
+  drop out of the ambassador list and must be re-nominated
+* ambassadors will be documented on the nodejs/ambassadors README.md
+
+The TSC may choose to remove an ambassador if things are not working
+out in terms of the content promoted and alignment with the project.
+
+Members of the Node.js organization can nominate ambassadors by opening
+a discussion issue with the nomination in
+[nodejs/collaborators](https://github.com/nodejs/collaborators/)
+titled `Nomination X to be an Ambassador`, where X is the name of the person
+being nominated.
+
+## Reviewing content
+
+Ambassadors are asked to share links to new content before asking
+for the content to be promoted. The goal is to provide the opportunity
+for collaborators to help improve content as opposed to a detailed
+nitpick review.
+
+This is done by opening an issue in the nodejs/ambassadors
+repository on which access is limited to members of the Node.js
+organization. The issue may include the link to existing published
+content or a draft of the content itself.
+
+Interested members of the Node.js organization may comment/make
+suggestions in the issue and the ambassadors are expected to
+consider those suggestions and update the content when they feel it
+is appropriate.
+
+Explicit approval is not required in the issue (i.e. if there is no
+input, that's OK), but objections should addressed before
+requesting promotion.
+
+## Expectations on Ambassadors
+
+Ambassadors are expected to engage both within the project and externally
+in a manner that is consistent with the project's
+[Code of Conduct](https://github.com/nodejs/admin/blob/main/CODE_OF_CONDUCT.md)
+as well as the additional expectations outlined in
+[MemberExpectations](https://github.com/nodejs/admin/blob/main/MemberExpectations.md)
+for a leadership position.
```

Comment:
So an ambassador is a leadership position but also it's totally fine if they have no connection to the project other than the ambassador role? (I do think they should be bound to the Member Expectations. I just don't think it should be because of anything having to do with leadership.)

Reply:
I did not intend to say it was a leadership position, but the working for MemberExpectations lists the expectations for leaders. I meant to say that the expectation on those in leadership positions as documented in MemberExpectations apply to abassadors. Not sure if you can think of a better way to express, versus implying an Abassador is a leadership positiion.

Reply:
Maybe just remove "for a leadership position"? I'll add a suggestion in another comment. It's not working right when I do it in this comment. (Maybe a change has already been made?)